dialog.go

```diff
@@ -22,15 +23,18 @@ type ConfirmationDialog struct {
 // The dialog is modal, so a user cannot interact other
 // Views until the user closes the dialog
 type SelectDialog struct {
-	View    *Window
-	result  int
-	value   int
-	rg      *RadioGroup
-	list    *ListBox
-	typ     SelectDialogType
-	onClose func()
+	View      *Window
+	btnResult int
+	value     int
+	rg        *RadioGroup
+	list      *ListBox
+	typ       SelectDialogType
+	onClose   func()
 }
 
+// a separate function to avoid api changes; assign to this function to create an editbox
+var createEditbox func(parent Control, width int, text string, scale int, dlg *ConfirmationDialog) *EditField
```

Comment:
What happens if a user creates a dialog with EditBox and then tries to create a dialog with ListBox? As far as I see, createEditbox function pointer is assigned after the first call and then the user is not able to create Dialog without EditBox? More thoughts in discussion

Reply:
Oh yes. You are right...

Then, I think we have no option but to change function API to get edit box parameter as input or create a separate function altogether. 
I did this so that we don't have to change function parameters.